bucket/quassel.json

```
@@ -0,0 +1,49 @@
+{
+    "description": "Modern, cross-platform, distributed IRC client based on the Qt framework.",
+    "homepage": "https://quassel-irc.org",
+    "license": "GPL-2.0|GPL-3.0",
```

```suggestion
    "license": "GPL-2.0-or-later|GPL-3.0-or-later",
```
Check **"Identifier"** at https://spdx.org/licenses/

https://github.com/quassel/quassel/blob/master/gpl-3.0.txt has `or any later versions` in it, therefore it is `GPL-3.0-or-later`.